src/app/vm/vm-sidebar/network-detail/nics/nic/nic-fields/nic-fields.component.html

```diff
@@ -1,19 +1,35 @@
-<div class="row">
-  <div>{{ 'VM_PAGE.NETWORK_DETAILS.ipAddress' | translate }}</div>
-  <div>{{ ipWithSuffix | translate }}</div>
-</div>
+<cs-parameters-pair
+  [name]="'VM_PAGE.NETWORK_DETAILS.ipAddress'"
+  [value]="ipWithSuffix | translate"
+></cs-parameters-pair>
+<cs-parameters-pair
+  *ngIf="nic?.macAddress"
```

Comment:
Can we get rid of Elvis operator (?.) and use *ngIf="nic" on top of the nic-fields-component?